doc/source/whatsnew/v0.20.0.txt

```diff
@@ -1348,6 +1348,7 @@ If indicated, a deprecation warning will be issued if you reference theses modul
     "pandas.parser", "pandas.io.libparsers", "X"
     "pandas.formats", "pandas.io.formats", ""
     "pandas.sparse", "pandas.core.sparse", ""
+    "pandas.tools", "pandas.core.tools", "pandas.tools.plotting"
```

Comment:
I don't think this will be clear. I would rather add another new line for the plotting 

```
"pandas.tools.plotting", "pandas.plotting", "X"
```